tests/other/test_material.py

```
@@ -0,0 +1,19 @@
+from django.test import TestCase
```

Ruff (0.8.2)

`django.test.TestCase` imported but unused

Remove unused import: `django.test.TestCase`

(F401)